skm-ice/skm-ice-stateviewer
# state-viewer displays string type data incorrectly

Before this commit strings were shown like this: [[[+]]]. The problem occurred after upgrading clojurescript from 1.9.229 to 1.9.946.

How to test it:
Press F12 in your application and check out how the strings in your state are displayed.
